plugins/vst_base/CMakeLists.txt

```diff
@@ -31,6 +31,17 @@ BUILD_PLUGIN(vstbase vst_base.cpp VstPlugin.cpp VstPlugin.h communication.h MOCF
 
 IF(LMMS_BUILD_LINUX AND NOT WANT_VST_NOWINE)
 
+SET(WINE_INCLUDES_AND_LIBS -I/usr/include/wine/windows)
+
+# Development
+IF(EXISTS "/opt/wine-devel/lib64/libwine.so")
+	SET(WINE_INCLUDES_AND_LIBS -I/opt/wine-devel/include -L/opt/wine-devel/lib)
+ENDIF()
+# Staging
+IF(EXISTS "/opt/wine-staging/lib64/libwine.so")
+	SET(WINE_INCLUDES_AND_LIBS -I/opt/wine-staging/include -L/opt/wine-staging/lib)
+ENDIF()
```

Comment:
`WINE_INCLUDE_DIR` gets value `/opt/wine-devel/include/wine` and `wine/*.h` headers cannot be found from there. Path needs to be `/opt/wine-devel/include`.

`WINE_LIBRARY` gets value `/opt/wine-devel/lib/libwine.so` and this cannot be linked. Path needs to be `/opt/wine-devel/lib`.